src/store/movie/action-creators.js

```diff
@@ -91,6 +90,7 @@ export const sortMoviesList = (sortByParam, orderByParam) => {
     return (dispatch) => {
         api.sortMovies(sortByParam, orderByParam).then((moviesData) => {
             dispatch(setSortedMoviesList(moviesData.data));
+            dispatch(setTotalAmount(moviesData.totalAmount));
```

Comment:
minor (so it's up to you whether to change it or not): I think setting `totalAmount` to global store is little bit excessive since we have movies in store and can calculate it (can't we? 🤔)
and take it in `mapStateToProps`

Reply:
thanks, agree